sdk/daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Upgrading.scala

```diff
@@ -629,6 +640,7 @@ case class TypecheckUpgrades(
       (existingDatatypes, _new) <- checkDeleted(
         unownedDts,
```

Comment:
Suggestion: use named arguments here, especially for the predicate.

Reply:
done